### PR TITLE
feat: add office video

### PR DESCRIPTION
**Pull request by AnjaMis**
This PR adds OKP4 office tour video.

### CODE REVIEW

**Review by MdechampG — APPROVED** [review]
Perfect Thank you !

**Review by ccamel — APPROVED** [review]
:+1:

File: i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Bureaux 🏢
+
+Nos bureaux sont situés au 1 Passage de l'Europe 31400 Toulouse - FRANCE
+
+8 places de parking sont réservées à OKP4.
+
+Notre filiale OKP4 Gmbh est située  
+Our subsidiary OKP4 GmbH is located
```
Comment (lolottetheclash) [inline i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx:14]: There is an extra translation here

File: i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+8 places de parking sont réservées à OKP4.
+
+Notre filiale OKP4 Gmbh est située  
+Our subsidiary OKP4 GmbH is located  
+Peter-Lühr-Straße 7  
+81739 Munich  
+GERMANY
+
+#### Jetez un coup d'œil à notre bureau de Toulouse !
```
Comment (lolottetheclash) [inline i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx:19]: To remain consistent with the French text above, I would have kept this translation:
#### Jetez un coup d'œil à nos bureaux de Toulouse !

File: i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx (hunk 1 above)
Comment (AnjaMis) [reply i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx]: Good catch! 🙈

File: i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx (hunk 2 above)
Comment (AnjaMis) [reply i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx]: Done!

**Review by lolottetheclash — APPROVED** [review]
Ok great !

**Pull request merged**